From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: wow wow <wow@wowow.com>
Date: Tue, 4 Jun 2024 22:12:37 -0500
Subject: [PATCH] Make the entity tracker track players every tick rather than
 every two ticks


diff --git a/src/main/java/net/minecraft/server/EntityTracker.java b/src/main/java/net/minecraft/server/EntityTracker.java
index cb72b36725927272fddfdad929b268a7b06140ac..3afa7fe3fbfbd845712fad77d424a906ef9df246 100644
--- a/src/main/java/net/minecraft/server/EntityTracker.java
+++ b/src/main/java/net/minecraft/server/EntityTracker.java
@@ -24,9 +24,10 @@ public class EntityTracker {
 
     public void track(Entity entity) {
         if (entity instanceof EntityPlayer) {
-            this.addEntity(entity, 512, 2);
+            // Track players every tick
+            this.addEntity(entity, 512, 1);
             EntityPlayer entityplayer = (EntityPlayer) entity;
-            Iterator iterator = this.c.iterator();
+            Iterator<EntityTrackerEntry> iterator = this.c.iterator();
 
             while (iterator.hasNext()) {
                 EntityTrackerEntry entitytrackerentry = (EntityTrackerEntry) iterator.next();
